data/names/ru.json

```diff
@@ -5870,7 +5870,624 @@
+      [ "Редман", "Redman" ],
+      [ "Рентерия", "Renteria" ],
+      [ "Риверс", "Rivers" ],
+      [ "Рорк", "Roark" ],
+      [ "Робисон", "Robison" ],
+      [ "Хорн", "Horn" ],
+      [ "Роудс", "Rhodes" ],
+      [ "Рокуэлл", "Rockwell" ],
+      [ "Ролз", "Rawls" ],
+      [ "Ропер", "Roper" ],
+      [ "Роуленд", "Rowland" ],
+      [ "Роша", "Rocha" ],
+      [ "Гроув", "Grove" ],
+      [ "Гроувз", "Groves" ],
+      [ "Роу", "Roe" ],
+      [ "Роял", "Royal" ],
+      [ "Рапп", "Rupp" ],
+      [ "Фиш", "Fish" ],
+      [ "Рэглэнд", "Ragland" ],
+      [ "Рэгсдэйл", "Ragsdale" ],
+      [ "Рэйми", "Ramey" ],
+      [ "Рэнкин", "Rankin" ],
+      [ "Рэтлифф", "Ratliff" ],
+      [ "Саймонс", "Simons" ],
+      [ "Сальдивар", "Saldivar" ],
+      [ "Салмон", "Salmon" ],
+      [ "Самора", "Zamora" ],
+      [ "Сэмюэл", "Samuel" ],
+      [ "Бонд", "Bond" ],
+      [ "Сэлф", "Self" ],
+      [ "Сэвидж", "Savage" ],
+      [ "Норт", "North" ],
+      [ "Сент-Джон", "St. John" ],
+      [ "Сильвер", "Silver" ],
+      [ "Сильвестр", "Sylvester" ],
+      [ "Сильвия", "Silvia" ],
+      [ "Стронг", "Strong" ],
+      [ "Синглтэри", "Singletary" ],
+      [ "Стэйплс", "Staples" ],
+      [ "Слэк", "Slack" ],
+      [ "Слотэр", "Slaughter" ],
+      [ "Слейтер", "Slater" ],
+      [ "Хёрд", "Heard" ],
+      [ "Смоллвуд", "Smallwood" ],
+      [ "Соарес", "Soares" ],
+      [ "Спэйн", "Spain" ],
+      [ "Спрэйг", "Sprague" ],
+      [ "Старнс", "Starnes" ],
+      [ "Стейли", "Staley" ],
+      [ "Стейнберг", "Steinberg" ],
+      [ "Стэйтон", "Staton" ],
+      [ "Гласс", "Glass" ],
+      [ "Уолл", "Wall" ],
+      [ "Сент-Клер", "Stclair" ],
+      [ "Стовалл", "Stovall" ],
+      [ "Стовер", "Stover" ],
+      [ "Столлингс", "Stallings" ],
+      [ "Стоукс", "Stokes" ],
+      [ "Пейдж", "Page" ],
+      [ "Стрейндж", "Strange" ],
+      [ "Стрэттон", "Stratton" ],
+      [ "Стэнфилд", "Stanfield" ],
+      [ "Сунига", "Zuniga" ],
+      [ "Сэй", "Seay" ],
+      [ "Сэйлор", "Saylor" ],
+      [ "Таварес", "Tavares" ],
+      [ "Таннер", "Tanner" ],
+      [ "Тейшейра", "Teixeira" ],
+      [ "Теллез", "Tellez" ],
+      [ "Тернер", "Turner" ],
+      [ "Титус", "Titus" ],
+      [ "Толберт", "Tolbert" ],
+      [ "Мерчант", "Merchant" ],
+      [ "Траммелл", "Trammell" ],
+      [ "Траутман", "Troutman" ],
+      [ "Труонг", "Truong" ],
+      [ "Трэхан", "Trahan" ],
+      [ "Тэкетт", "Tackett" ],
+      [ "Тэм", "Tam" ],
+      [ "Бэйлс", "Bales" ],
+      [ "Тёрпин", "Turpin" ],
+      [ "Уайлс", "Wiles" ],
+      [ "Уиллер", "Wheeler" ],
+      [ "Уиллингем", "Willingham" ],
+      [ "Уиллоби", "Willoughby" ],
+      [ "Уинн", "Winn" ],
+      [ "Уиттен", "Whitten" ],
+      [ "Уиттингтон", "Whittington" ],
+      [ "Смарт", "Smart" ],
+      [ "Уоркмен", "Workman" ],
+      [ "Уотерман", "Waterman" ],
+      [ "Уэбстер", "Webster" ],
+      [ "Уэллетт", "Ouellette" ],
+      [ "Уэлч", "Welch" ],
+      [ "Фалкон", "Falcon" ],
```

Comment:
This one is really two surnames: one English and one Spanish. For the English one, it would be `Фэлкон`, like [`Джона Фэлкон`](https://ru.wikipedia.org/wiki/%D0%A4%D1%8D%D0%BB%D0%BA%D0%BE%D0%BD,_%D0%94%D0%B6%D0%BE%D0%BD%D0%B0). For the Spanish one, it would be `Фалькон`, like [`Хуан Фалькон`](https://ru.wikipedia.org/wiki/%D0%A4%D0%B0%D0%BB%D1%8C%D0%BA%D0%BE%D0%BD,_%D0%A5%D1%83%D0%B0%D0%BD_%D0%9A%D1%80%D0%B8%D1%81%D0%BE%D1%81%D1%82%D0%BE%D0%BC%D0%BE).

Both make sense within the context of the game, given the large Spaniard population of the US. In fact, [the page for the surname on Wikipedia](https://en.wikipedia.org/wiki/Falcon_(surname)) has 7/16 bearers be of Spanish descent (and one of French descent, with their surname being pronounced like the Spanish one, so maybe that counts?). I'm leaning towards the Spanish one also because it makes the list a bit more-interesting and a bit more-inclusive.